helm_helpers.go

```diff
@@ -614,7 +619,7 @@ func cleanUntrackedReleases() {
 	} else {
 		for ns, releases := range toDelete {
 			for r := range releases {
-				if r.isReleaseConsideredToRun() {
+				if !r.isReleaseConsideredToRun() {
```

Comment:
Why this inversion? AFAIK this says "did you specifically tell me to manage this release via startup flags?". So we're now saying "if we don't manage this release, don't delete it", and below, "otherwise, the flags were populated with this release, we do manage this release -- delete it". Doesn't that mean this will only delete releases we explicitly include on startup in `target` or `group`?

Reply:
this was another bug, `r.isReleaseConsideredToRun()` basically means "is this release being managed" and if this condition is true we skip the release, in this case we want to skip the release if it's not being managed.

Reply:
Got it. Thanks.